src/app/components/modals/courseInfoModal/CourseInfoModal.tsx

```diff
@@ -258,16 +258,10 @@ export default function CourseInfoModal({
               <Typography component="p">{prerequisites(data)}</Typography>
               <Typography component="p">GE: {ge(data)}</Typography>
               <Skeleton loading={enrollLoading}>
-                <div className="flex flex-wrap items-center gap-2">
-                  <Typography component="p">Quarters Offered:</Typography>
-                  {enrollmentInfo!.map((e, i) => {
-                    return (
-                      <Chip key={i} color={getQuarterColor(e.term.title)}>
-                        {e.term.title} {e.term.catalogYear}, {e.instructor}
-                      </Chip>
-                    );
-                  })}
-                </div>
+                <Typography component="p">Quarters Offered:</Typography>
```

Comment:
Move this into the `QuartersOfferedTable` component since some courses might not have loaded enrollment information and this text will still appear